### ISSUE TITLE
groupbytraceprocessor: TestErrorOnProcessResourceSpansContinuesProcessing is unstable

### ISSUE THREAD

**Issue by tigrannajaryan** [open-telemetry/opentelemetry-collector#1923]
See https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-collector/3868/workflows/6cbedc3f-98dd-439d-ae9f-987aa9e3e8e2/jobs/42433

```
2020-10-08T16:32:15.916Z	WARN	groupbytraceprocessor/processor.go:133	failed to process batch	{"error": "couldn't add spans to new trace: some unexpected error", "traceID": "01020304"}
go.opentelemetry.io/collector/processor/groupbytraceprocessor.(*groupByTraceProcessor).processResourceSpans
	C:/Users/circleci/project/processor/groupbytraceprocessor/processor.go:133
go.opentelemetry.io/collector/processor/groupbytraceprocessor.TestErrorOnProcessResourceSpansContinuesProcessing
	C:/Users/circleci/project/processor/groupbytraceprocessor/processor_test.go:708
testing.tRunner
	c:/go/src/testing/testing.go:1108
FAIL
FAIL	go.opentelemetry.io/collector/processor/groupbytraceprocessor	0.728s
```

**Comment by tigrannajaryan** [comment]
@jpkrohling can you please have a look?

**Comment by jpkrohling** [comment]
This one should also be moved to -contrib, please.

**Comment by jpkrohling** [comment]
Actually, this should be closed, as it's a duplicate of #1927. The error in the test log isn't unexpected, it's actually part of the test:

```
2020-10-08T16:32:15.916Z	WARN	groupbytraceprocessor/processor.go:133	failed to process batch	{"error": "couldn't add spans to new trace: some unexpected error", "traceID": "01020304"}
go.opentelemetry.io/collector/processor/groupbytraceprocessor.(*groupByTraceProcessor).processResourceSpans
	C:/Users/circleci/project/processor/groupbytraceprocessor/processor.go:133
go.opentelemetry.io/collector/processor/groupbytraceprocessor.TestErrorOnProcessResourceSpansContinuesProcessing
	C:/Users/circleci/project/processor/groupbytraceprocessor/processor_test.go:708
testing.tRunner
	c:/go/src/testing/testing.go:1108
```

If we scroll up the test logs from the link in the issue description, you'll see this:

```
--- FAIL: TestPeriodicMetrics (0.04s)
    event_test.go:411: 
        	Error Trace:	event_test.go:411
        	            				event_test.go:366
        	Error:      	Not equal: 
        	            	expected: int(0)
        	            	actual  : float64(1)
        	Test:       	TestPeriodicMetrics
```

So, the actual action to take on this issue here is to enable back the test :-)
